### PR TITLE
Minecraft PC 1.13.2-pre2 & 1.13.2 support

### PR DESCRIPTION
**Pull request by ellisgl**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea
```
Comment (plexigras) [inline .gitignore]: this again?

> Please add that in your global ignore file or in .git/info/exclude and remove it from the pr.

Comment (ellisgl) [reply .gitignore]: In the next commit (63a808d), it was removed.

**Comment by rom1504** [comment]
looks good, thanks

**Comment by rom1504** [comment]
released as minecraft-data 2.34.0, you can run npm update to get support in node-minecraft-protocol

**Comment by ellisgl** [comment]
If I had more time I would look at getting 1.13.x in Primarine-Chunk...

**Pull request merged**